### PR TITLE
Rule for throttle

### PR DESCRIPTION
**Pull request by cupofjoakim**
As discussed in #235. I wasn't really sure on the message in the rule, we did say that we should point it to the documentation but I think we could work on the wording.

### CODE REVIEW

**Comment by cupofjoakim** [comment]
@cjtaylor1990 @stevemao Here's the PR!

File: lib/rules/rules.json [hunk 1]
```diff
@@ -272,5 +272,9 @@
     "flatten": {
     	"compatible": true,
     	"alternative": "Array.prototype.reduce((a,b) => a.concat(b), [])"
+    },
+    "throttle": {
+        "compatible": true,
+        "alternative": "Example of native implementation: https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_throttle"
     }
-}
```
Comment (stevemao) [inline lib/rules/rules.json]: Could you revert the change of this line? Otherwise LGTM. We should standardise link to the readme.

**Comment by cupofjoakim** [comment]
@stevemao Done! :)

**Comment by cupofjoakim** [comment]
@cjtaylor1990 Any chance you could give this a look? :)

**Comment by cjtaylor1990** [comment]
This looks great. Thanks for the PR, and sorry I've been MIA.
